### ISSUE TITLE
STRM64 aborting when built from Source

### ISSUE THREAD

**Issue by Retro-Hax** [gheskett/STRM64#2]
i tried both the binary package under Releases giving me an aborted (core dump) same happened compiling from source tho it also gave me an invalid fastbin error

coredumpctl:
```
                Stack trace of thread 1884921:
                #0  0x00007f5fcf6a164c n/a (libc.so.6 + 0x8864c)
                #1  0x00007f5fcf651938 raise (libc.so.6 + 0x38938)
                #2  0x00007f5fcf63b53d abort (libc.so.6 + 0x2253d)
                #3  0x00007f5fcf6957ee n/a (libc.so.6 + 0x7c7ee)
                #4  0x00007f5fcf6ab3dc n/a (libc.so.6 + 0x923dc)
                #5  0x00007f5fcf6ad590 n/a (libc.so.6 + 0x94590)
                #6  0x00007f5fcf6af37b n/a (libc.so.6 + 0x9637b)
                #7  0x00007f5fcf6af9da n/a (libc.so.6 + 0x969da)
                #8  0x00007f5fcf6b0f43 posix_memalign (libc.so.6 + 0x97f43)
                #9  0x00007f5fcfc4e696 av_malloc (libavutil.so.57 + 0x4e696)
                #10 0x00007f5fcfc2e2c3 av_frame_alloc (libavutil.so.57 + 0x2e2c3)
                #11 0x00005582cd10a19a n/a (/home/retro-hax/Retro-Hax/RomHacks/SM64Decomp/Tools/STRM64/STRM64 + 0x2919a)
                ELF object binary architecture: AMD x86-64
```

and here is the command i tried
```
./STRM64 Mountain_Village.mp3 -o 03_level_grass -l 1
invalid fastbin entry (free)
Aborted (core dumped)
```

**Comment by gheskett** [comment]
Hmm, interesting. I assume you have ffmpeg installed as well to handle the mp3 conversion? (Sorry if that bit wasn't explained very well by the README.)

**Comment by Retro-Hax** [comment]
Yes i do have ffmpeg installed

**Comment by gheskett** [comment]
Judging by the stack trace, it looks like there's probably some issue with the ffmpeg libraries that vgmstream is using. Given that it still failed when you built it yourself, I worry there could be some (CPU architectural?) issues with the vgmstream library being linked (which is the worst case scenario; getting that to cross compile was a headache and a half).

Does it work for you if you convert the mp3 into a wav file manually and pass that instead?

**Comment by gheskett** [comment]
Feel free as well to share the mp3 file you used so I can verify whether or not it works on my own machine. If it does, it'll unfortunately be a pretty difficult thing for me to debug.

**Comment by Retro-Hax** [comment]
About the conversion well i already tried the .wav file and well same issue sadly and i just tried the new uh released binary (0.1.2) and with the binary on the release page everything works with no problems using the same .mp3 file but still it is quite weird that i cant use the binary i built from source

**Comment by gheskett** [comment]
Glad to hear the new release worked for you, though it definitely is a little strange that you weren't able to get it working when building the source. Even if your source were to be out of date, I would have expected it to build/function for you just fine 🤔
